hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileBlock.java

```diff
@@ -449,7 +449,7 @@ public int getOnDiskSizeWithHeader() {
   }
 
   /** Returns the on-disk size of the data part + checksum (header excluded). */
-  int getOnDiskSizeWithoutHeader() {
+  public int getOnDiskSizeWithoutHeader() {
```

Comment:
Why do we need to change this modifier? Seems we do not call in the newly added tests?

Reply:
Yes, not used on tests anymore. Removing...